user_manual/android_app.rst

```diff
@@ -239,6 +251,42 @@ Just ask them to copy and paste and send it to you.
 .. figure:: images/android-14.png
    :alt: Federated share creation.
    
+Working With Multiple Files
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Android application can perform some operations on multiple files simultaneously, such as refreshing and deleting. 
+To select multiple files, long select the first file that you want to work with; you will see a checkbox appear on the far right-hand side. 
+After that, check the checkbox next to all the other files that you want to perform the same operation on, and then perform the operation.
+
+.. figure:: images/android-select-multiple-files.png
```

Comment:
I guess this screenshot is not ready yet

Reply:
Link does not work

Reply:
Link now fixed.

Reply:
Link fixed.

Reply:
👍